packages/ml-kit/README.md

```diff
@@ -0,0 +1,156 @@
+
+- `ModelNameResolver: () => ModelOptions`
+
+Useful to change the options used by the model of a task at runtime.
+
+**Tasks output events:**
+
+- [`{classificationAim}Predicted`](#events)
+- [`{regressionAim}Predicted`](#events)
+
+> Example usage in the application task graph:
+> 
+> ```typescript
+> on('inputDataForHumanActivity', run('humanActivityClassificationUsingCNN'));
+> on('humanActivityPredicted', run('writeRecords'));
+> 
+> on('inputDataStressLevel', run('stressLevelRegression'));
+> on('stressLevelPredicted', run('writeRecords'));
+> ```
+> 
+> **Note**: To use the `writeRecords` task, the persistence package must be installed and configured. See [persistence package docs](https://github.com/GeoTecINIT/awarns-framework/blob/main/packages/persistence/README.md).
+
+### Events
+
+| Name                           | Payload                             | Description |
+|--------------------------------|-------------------------------------|-------------|
+| `{classificationAim}Predicted` | [`Classification`](#classification) |             |
+| `{regressionAim}Predicted`     | [`Regression`](#regression)         |             |
+
+
+### Records
+
+#### Classification
+
+| Property               | Type                                            | Description                                                |
+|------------------------|-------------------------------------------------|------------------------------------------------------------|
+| `id`                   | `string`                                        | Record's unique id.                                        |
+| `type`                 | `string`                                        | Always `classification-prediction`.                        |
+| `change`               | `Change`                                        | Always `NONE`.                                             |
+| `timestamp`            | `Date`                                          | The local time when the classification record was created. |
+| `classificationResult` | [`ClassificationResult`](#classificationresult) | Object containing the results of the classification.       |
+
+##### ClassificationResult
+
+| Property     | Type                         | Description                                                                                                                |
+|--------------|------------------------------|----------------------------------------------------------------------------------------------------------------------------|
+| `prediction` | `ClassificationPrediction[]` | Array of the classification predictions generated by the model, containing a `label` and classification `score` per class. |
+| `timestamp`  | `Date`                       | The local time when the classification result was computed.                                                                |
```

Comment:
What's the difference between this timestamp and the top-level record timestamp? Perhaps they're redundant?